README.md

```diff
@@ -57,28 +59,106 @@ systemctl restart jans-auth
 
 ### Deployment
```

Comment:
I really don't even know why you have a deployment section at all... this should be in Jans docs.